### PR TITLE
[EH] Emulator prep for release

### PR DESCRIPTION
**Pull request by l0lawrence**

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
+## 5.11.8 (2024-05-16)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added support for EventHub Emulator
```
Comment (l0lawrence) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:7]:
```suggestion
- Added support for non-tls connections
```

File: sdk/eventhub/azure-eventhub/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 5.11.8 (Unreleased)
+## 5.11.8 (2024-05-16)
```
Comment (l0lawrence) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:3]:
```suggestion
## 5.12.0 (2024-05-16)
```

**Review by kashifkhan — APPROVED** [review]

**Review by swathipil — APPROVED** [review]

**Comment by l0lawrence** [comment]
just ran tests - all passed and updated changelog

**Pull request merged**